pyvista/core/filters/data_set.py

```diff
@@ -1110,6 +1111,156 @@ def outline_corners(  # type: ignore[misc]
         _update_alg(alg, progress_bar=progress_bar, message='Producing an Outline of the Corners')
         return wrap(alg.GetOutputDataObject(0))
 
+    def rescale(  # type: ignore[misc]
```

Comment:
```suggestion
    def resize(  # type: ignore[misc]
```

If we accept https://github.com/pyvista/pyvista/pull/7092 and add the `size` property, then we could rename this accordingly to better align their use

Reply:
This filter is reworked in #7726. In that PR, the `scale_factors` keyword is changed to `size`. In contrast to the  `scale` filter, where a scale factor is simply applied, `resize` uses the `size` parameter to _compute_ a new scale factor such that the mesh with have the desired size. I think this is the key distinction between these two filters.

If we do `out = mesh.resize(size=(1.0, 2.0, 3.0))`, then `out.size` returns `(1.0, 2.0, 3.0)`. It would be good to consider reviewing https://github.com/pyvista/pyvista/pull/7092 before this PR so that we can make use of the `size` parameter as part of the docstrings and show this relationship clearly.